.github/workflows/ci-test.yaml

```diff
@@ -0,0 +1,132 @@
+name: Build and Run Tests
+
+on:
+#   # schedule:
+#     # - cron: '0 5 * * *'  #Runs  daily at 5 AM UTC
+  push:
+    branches:
+    - CV2-5258-run-linter-Check-web-github-actions
+
+
+env:
+  GITHUB_BRANCH: ${{ github.head_ref || github.ref_name }}
+  GITHUB_JOB_NAME: ${{ github.job }}
+  GITHUB_COMMIT_MESSAGE: ${{ github.event.head_commit.message }}
+  NGROK_AUTH: ${{ secrets.NGROK_AUTH }}
+  AWS_ENDPOINT: ${{ secrets.AWS_ENDPOINT }}  
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }} 
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }} 
```

Comment:
The variables that are not needed to linter and unit tests can be left out of this PR, for simplicity sake.

Reply:
ok

Reply:
It fails now when I removed the line.

Reply:
https://github.com/meedan/check-web/actions/runs/10834511792/job/30063875381?pr=2097

Reply:
Nice!